views/address.tmpl

```diff
@@ -174,8 +268,8 @@
                       >
                           <input type="button" class="chart-size btn btn_sm year" value="Year" name="year">
                           <input type="button" class="chart-size btn btn_sm month" value="Month" name="month">
-                          <input type="button" class="chart-size btn btn_sm week btn-active" value="Week" name="week">
-                          <input type="button" class="chart-size btn btn_sm day" value="Day" name="day">
+                          <input type="button" class="chart-size btn btn_sm week" value="Week" name="week">
+                          <input type="button" class="chart-size btn btn_sm day btn-active" value="Day" name="day">
                           <input type="button" class="chart-size btn btn_sm all" value="All Blocks" name="all">
```

Comment:
I was thinking about the "All Blocks" button for the bar/group size, and I think that "Block" is more consistent with "Year", "Month", etc. for describing the width of a bin.  Might as well make that change in this PR too, if you agree with that.